documentation/src/main/java/testing/MyTest.java

```diff
@@ -8,9 +8,9 @@
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 
-import io.smallrye.reactive.messaging.providers.connectors.InMemoryConnector;
-import io.smallrye.reactive.messaging.providers.connectors.InMemorySink;
-import io.smallrye.reactive.messaging.providers.connectors.InMemorySource;
+import io.smallrye.reactive.messaging.in.memory.InMemoryConnector;
```

Comment:
I'm not sure I like the `in.memory`, maybe just `memory`?

Reply:
👍